src/runtime/gravity_core.c

```diff
@@ -1645,6 +1647,83 @@ static bool system_exit (gravity_vm *vm, gravity_value_t *args, uint16_t nargs,
 	RETURN_NOVALUE();
 }
 
+// MARK: - Date -
+
+static bool date_time (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	int t = (int)time(NULL);
+	RETURN_VALUE(VALUE_FROM_INT(t), rindex);
+}
+
+static bool date_second (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	int tm = localtime(&t)->tm_sec;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_minute (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	int tm = localtime(&t)->tm_min;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_hour (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	int tm = localtime(&t)->tm_hour;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_month_day (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	int tm = localtime(&t)->tm_mday;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_month (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	// Plus 1 because tm_mon is a base 0 number
+	int tm = localtime(&t)->tm_mon + 1;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_year (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	// Plus 1900 because tm_year is the number of years since 1900
+	int tm = localtime(&t)->tm_year + 1900;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_week_day (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	// Plus 1 because tm_wday is a base 0 number
+	int tm = localtime(&t)->tm_wday + 1;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_year_day (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	// Plus 1 because tm_wday is a base 0 number
+	int tm = localtime(&t)->tm_yday + 1;
+	RETURN_VALUE(VALUE_FROM_INT(tm), rindex);
+}
+
+static bool date_daylight_savings (gravity_vm *vm, gravity_value_t *args, uint16_t nargs, uint32_t rindex) {
+	#pragma unused(args,nargs)
+	time_t t = time(NULL);
+	// Plus 1 because tm_wday is a base 0 number
+	int tm = localtime(&t)->tm_isdst;
```

Comment:
You forgot about the `int tm = localtime(&t)->tm_isdst + 1;` or removing the comment